### PR TITLE
Create print-pooler-service-suspicious-file-creation.yaml

### PR DESCRIPTION
**Pull request by BlackB0lt**
Required items, please complete
   
   Change(s):
   - Create New rule to detect DLL Load via PrintNightmare Vulnerability

   Reason for Change(s):
   - New rule for DLL Load detection on Spooler Service backup folder

   Version Updated:
   - Required only for Detections/Analytic Rule templates

   Testing Completed:
   - Tested on event with sigma rule

   Checked that the validations are passing and have addressed any issues that are present:
   - See guidance below

References: https://github.com/SigmaHQ/sigma/blob/master/rules/windows/image_load/image_load_spoolsv_dll_load.yml

### CODE REVIEW

File: Hunting Queries/Microsoft 365 Defender/Exploits/print-pooler-service-suspicious-file-creation.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+id: 67309406-12ad-4591-84db-0cc331634d0c
+name: Windows Spooler Service Suspicious File Creation
+description: |
+  The query digs in Windows print spooler drivers folder for any file creations,
+  MANY OF THE FILES THAT SHOULD COME UP HERE MAY BE LEGIT. Suspicious DLL is load from Spooler Service backup folder. 
+  This behavior is used from PoC Exploit of CVE-2021-34527, CVE-2021-1675 or CVE-2022-21999.
+requiredDataConnectors:
+- connectorId: Microsoft365Defender
```
Comment (aprakash13) [inline Hunting Queries/Microsoft 365 Defender/Exploits/print-pooler-service-suspicious-file-creation.yaml:8]: Please use MicrosoftThreatProtection  as ConnectorId

File: Hunting Queries/Microsoft 365 Defender/Exploits/print-pooler-service-suspicious-file-creation.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  dataTypes:
+  - DeviceFileEvents
+tactics:
+- Privilege escalation
+- Lateral movement
+- Exploit
```
Comment (aprakash13) [inline Hunting Queries/Microsoft 365 Defender/Exploits/print-pooler-service-suspicious-file-creation.yaml:14]: Please add relevantTechnique here.

**Review by aprakash13 — CHANGES_REQUESTED** [review]

**Comment by shainw** [comment]
@BlackB0lt - Thanks for the contribution! Please fill out the PR template and remove the guidance section per instructions.  Ajeet has also made a few comments for you to fix up.

**Comment by vmanojreddy** [comment]
@BlackB0lt please update the requested changes ,thanks!!!

**Comment by vmanojreddy** [comment]
@BlackB0lt please update the requested changes ,thanks!!!

**Comment by BlackB0lt** [comment]
> @BlackB0lt please update the requested changes ,thanks!!!

Completed PR.

**Comment by vmanojreddy** [comment]
@BlackB0lt In Check list cla is failing. can you please check and update it. thanks!!!

**Comment by vmanojreddy** [comment]
@aprakash13  Please review and approve the request changes. thanks!!!

**Comment by BlackB0lt** [comment]
> @BlackB0lt In Check list cla is failing. can you please check and update it. thanks!!!

I am authorized cla application. How to fix cla issue? This 1st PR on Azure Sentinel.

**Comment by aprakash13** [comment]
@BlackB0lt :  It was suggested to use MicrosoftThreatProtection as ConnectorId as well as add relevantTechnique field to the query. I see that they have been marked as resolved but for some reason I do not see that updated in the file. Please let me know if I am missing something here.

**Comment by BlackB0lt** [comment]
> 

I added relevantTechniques

**Review by aprakash13 — APPROVED** [review]
Thanks @BlackB0lt

**Pull request merged**